code-reviews/example1/styles.css

```diff
@@ -0,0 +1,15 @@
+.main {
```

Comment:
should you add default styling using * selector?

Reply:
Can the units be more specified ie units for properties like margin, padding, width, etc. This ensures proper rendering across different browsers.?